lib/chewy/type/import.rb

```diff
@@ -84,19 +84,26 @@ def bulk_body(action_objects, indexed_objects = nil)
           end
         end
 
+        # Returns the value of the identifier for an object
```

Comment:
@pyromaniac do you think it makes sense to call out from here, into Type.rb to avoid the code duplication? Alternatively I could remove the class checking from Type.rb .identify and just leave it here.

I wasn't quite clear on your instructions regarding this point.

Also there are definitely existing specs that cover Type.rb .identify during which root_object is nil